apps/astarte_realm_management/lib/astarte_realm_management/queries.ex

```diff
@@ -209,18 +209,31 @@ defmodule Astarte.RealmManagement.Queries do
   end
 
   def check_astarte_health(client, consistency) do
-    realms_count_statement = """
-    SELECT COUNT(*)
+    query_schema = """
+      SELECT count(value)
+      FROM astarte.kv_store
+      WHERE group='astarte' AND key='schema_version'
+    """
+
+    query_realms = """
```

Comment:
See above (this should be `realms_statement`)

Reply:
Also, add a comment to explain why that realm name was chosen so that we can remind our future selves the next time we end up here